automation/check-copyright.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import logging
+import re
+import sys
+from git import Repo
+from datetime import date
+
+logging.basicConfig(level=logging.INFO, format='[%(levelname)s] %(message)s')
+log = logging.getLogger(__name__)
+
+COPYRIGHT_RE = re.compile(
+    "# Copyright.* ([0-9]{4}(?:-[0-9]{4})?) .*Red Hat, Inc.")
```

Comment:
What if the contributor is not from Red Hat?

Reply:
True, I did not consider that (in a quick grep all files that I saw were `Red Hat, Inc.` so I just used it in the regex). It should end with just `.*` then.